tests/foreman/cli/test_remoteexecution.py

```diff
@@ -511,6 +511,13 @@ def test_positive_run_receptor_installer(self):
 
         :BZ: 1818076
         """
+        # Set Host parameter source_display_name to something random.
+        # To avoid 'name has already been taken' error when run multiple times
+        # on a machine with the same hostname.
+        host_id = Host.info({'name': settings.server.hostname})['id']
+        Host.set_parameter({'host-id': host_id, 'name': 'source_display_name',
+                            'value': gen_string('alpha')})
```

Comment:
black doesn't like this syntax and fails travis. See travis log for details.
Please fix.